Formula/fish.rb

```diff
@@ -39,7 +54,30 @@ def caveats; <<-EOS.undent
     EOS
   end
 
+  def post_install
+    (share/"fish/vendor_functions.d").mkpath
```

Comment:
Yeah we can't use `pkgshare` instead of `"#{HOMEBREW_PREFIX}/share/fish"` but this `post_install` method is installing to `share/"fish"` which _is_ `pkgshare`.